pom-modify/modify-project.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [[ $1 == "" ]]; then
+    echo "arg1 - the path to the project, where high-level pom.xml is"
```

Comment:
Maybe say "the path to the project, the directory where high-level pom.xml is"

Reply:
Done